test_unstructured_inference/models/test_yolox.py

```diff
@@ -0,0 +1,213 @@
+import os
+import shutil
+
+import jsons
+import pytest
+from fastapi.testclient import TestClient
+
+from unstructured_inference import api
+from unstructured_inference.inference.layout import DocumentLayout
+from unstructured_inference.models.yolox import yolox_local_inference, _get_model_loading_info
+from unstructured_inference.models import UnknownModelException
+
+
+@pytest.mark.skipif(
```

Comment:
Pytest has a pattern that I think fits better here. If you decorate the long-running tests with something like `@pytest.mark.long`, then in the make target for only short running tests you can run `pytest -m "not long" test_${PACKAGE_NAME} --cov=${PACKAGE_NAME} --cov-report term-missing`. See [the pytest markers documentation](https://docs.pytest.org/en/latest/example/markers.html) for more examples and explanation.

Reply:
Just changed it to that way, it's much cleaner. Thanks!